src/groupeddataframe/grouping.jl

```diff
@@ -66,6 +72,11 @@ See the following for additional split-apply-combine operations:
 * `map` : apply a function to each group of a `GroupedDataFrame` (without combining)
 * `combine` : combine a `GroupedDataFrame`, optionally applying a function to each group
 
+`GroupedDataFrame` also supports the dictionary interface, where its keys are
+`NamedTuple`s containing the values of the grouped columns. You may also index
+using a `Tuple`, in which case the values should be in the same order as the `cols`
```

Comment:
`Tuple` indexing is non-standard, but I am OK to allow it as it might be convenient. @nalimilan - any thoughts on this?

Reply:
I'm not sure there's a strong use case for it either, but it doesn't sound too problematic, so...

Reply:
I am OK with `Tuple` now.